pyomo/contrib/pynumero/tests/test_cyipopt_examples.py

```diff
@@ -37,18 +37,17 @@
         "Pynumero needs the ASL extension to run CyIpopt tests")
 
 import pyomo.contrib.pynumero.algorithms.solvers.cyipopt_solver as cyipopt_solver
-if not cyipopt_solver.ipopt_available:
-    raise unittest.SkipTest("PyNumero needs CyIpopt installed to run CyIpopt tests")
+if not cyipopt_solver.cyipopt_available:
+    raise unittest.SkipTest(
+        "PyNumero needs CyIpopt installed to run CyIpopt tests")
 import cyipopt as cyipopt_core
 
 example_dir = os.path.join(this_file_dir(), '..', 'examples')
 
 class TestPyomoCyIpoptSolver(unittest.TestCase):
     def test_status_maps(self):
-        self.assertEqual(len(cyipopt_core.STATUS_MESSAGES),
```

Comment:
Why remove them?

Reply:
We now have a case where we are mapping more descriptions than a specific version of cyipopt defines (we hold mappings for 1.0.3 and <= 0.3.0).  So, our map will always be bigger.  It is sufficient to catch that we can map all the statuses that this cyipopt version returns.